### PR TITLE
Add base config for Next.js

### PR DESCRIPTION
**Pull request by phuctm97**
Add base config for Next.js, source is the default tsconfig.json generated by Next CLI.

### CODE REVIEW

**Comment by orta** [comment]
👍🏻

File: bases/next.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Next.js",
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": false,
```
Comment (aladdin-add) [inline bases/next.json:9]: can I ask why "strict" => false; it seems a good practice to enable it?

Comment (orta) [reply bases/next.json:9]: That's what next folks do, it's there call but it's not unreasonable to me - not every project needs to be in strict mode

Comment (frencojobs) [reply bases/next.json:9]: Strict mode was turned off in favor of people who might not be comfortable with TypeScript but still want to have the best IntelliSense experience. In their documentation, they actually recommend to turn on strict mode when comfortable.

frenco's screenshot

I think there should also be a next + strictest option. What do you think @orta?

Comment (orta) [reply bases/next.json:9]: I think we're fine without, if they default to not strict then having strict on is a _project_ specific change and making it easy to see that choice in the `tsconfig.json` is a win. If it came from here, people might think its a next-js made call.

Comment (frencojobs) [reply bases/next.json:9]: Yea that makes sense, but I went ahead and made a PR. You can merge it if you want, or close it if you will.

**Pull request merged**